tests/server.py

```diff
@@ -899,6 +906,22 @@ def setup_test_homeserver(
             "args": {"database": test_db_location, "cp_min": 1, "cp_max": 1},
         }
 
+        # Check if we have set up a DB that we can use as a template.
+        global PREPPED_SQlITE_DB_CONN
```

Comment:
```suggestion
        global PREPPED_SQLITE_DB_CONN
```

what are you doing :D

Reply:
What's the difference there?

Reply:
hahaha